doc/guides/backporting-to-release-lines.md

```diff
@@ -31,9 +31,9 @@ release line. All commands will use the `v10.x-staging` branch as the target
 branch. In order to submit a backport pull request to another branch, simply
 replace that with the staging branch for the targeted release line.
 
-1. Checkout the staging branch for the targeted release line
-2. Make sure that the local staging branch is up to date with the remote
-3. Create a new branch off of the staging branch
+1. Checkout the staging branch for the targeted release line.
```

Comment:
While we're in here, maybe this too?

```suggestion
1. Check out the staging branch for the targeted release line.
```

Reply:
Are we sure we'd want to change this considering it's in reference to the git `checkout` command?

Reply:
I’ll go ahead with landing this as this PR hasn’t modified that part.